app/support/route.ts

```diff
@@ -1,3 +1,11 @@
+import { service } from '@ember-decorators/service';
 import Route from '@ember/routing/route';
 
-export default class Support extends Route {}
+export default class Support extends Route {
+    @service analytics;
+    actions = {
```

Comment:
Probably should use the `@action` decorator or move this up to the `.extends({})` mixin